package_bag/routines.py

```diff
@@ -20,12 +20,20 @@ class BagDiscoverer(object):
     Validates bag structure and bag info file, renames bag with unique ID
     """
 
+    def __init__(self):
+        self.src_dir = settings.SRC_DIR
+        self.tmp_dir = settings.TMP_DIR
+        for dir in [join(settings.BASE_DIR, self.src_dir),
+                    join(settings.BASE_DIR, self.tmp_dir)]:
```

Comment:
So I'm a little confused about this because I don't see you joining the base dir to these directories in any of the methods below? Am I missing something here, or should you just be checking if `settings.SRC_DIR` and `settings.BASE_DIR` exist?

Reply:
This was an oversight from using an example--I've fixed it, thank you!